apps/src/applab/firebaseCounters.js

```diff
@@ -76,16 +61,21 @@ export function updateTableCounters(tableName, rowCountChange, updateNextId) {
  *   limits is exceeded.
  */
 export function incrementRateLimitCounters() {
-  return loadConfig().then(() => getCurrentTime())
-  .then(currentTimeMs => {
-    let promises = [];
-    Object.keys(rateLimitMap).forEach(interval => {
+  let promises = [];
+  promises.push(loadConfig());
+  promises.push(getCurrentTime());
+  return Promise.all(promises).then(results => {
+    const config = results[0];
+    const currentTimeMs = results[1];
```

Comment:
Ah, yes! I had tried that, but not found the right syntax for doing it with const.